src/util/RandomEvictionCache.h

```diff
@@ -6,6 +6,7 @@
 #include "util/Math.h"
 #include "util/NonCopyable.h"
 
+#include <functional>
```

Comment:
There's a use of std::function in the file. It was only compiling before because all the places that used it happened to include <functional> before hand. I can add a comment to the commit or make it a separate commit if you like.